src/System.Net.Requests/tests/AuthenticationManagerTest.cs

```diff
@@ -29,11 +29,30 @@ public void Register_Null_Throws()
         [Fact]
         public void Register_Unregister_ModuleCountUnchanged()
         {
-            int initialCount = GetModuleCount();
-            IAuthenticationModule module = new CustomModule();
-            AuthenticationManager.Register(module);
-            AuthenticationManager.Unregister(module);
-            Assert.Equal(initialCount, GetModuleCount());
+            RemoteInvoke(() =>
+            {
+                int initialCount = GetModuleCount();
```

Comment:
GetModuleCount needs to be made static.  Otherwise this is closing over `this`, which isn't supported with RemoteInvoke.